inbox/util/addr.py

```diff
@@ -46,7 +46,9 @@ def parse_mimepart_address_header(mimepart, header_name):
     # Consult RFC822 Section 6.1 and RFC2047 section 5 for details.
     addresses = set()
     for section in mimepart.headers._v.getall(normalize(header_name)):
-        for phrase, addrspec in rfc822.AddressList(section).addresslist:
+        for phrase, addrspec in email.utils.getaddresses([section]):
+            if not addrspec:
```

Comment:
The behavior would change here if your `section` is `"x" <>`. As in, to maintain compatibility, you'd want this line to be `if not addrspec and not phrase:`. I'm pretty sure just skipping empty addresses as you're doing here is fine as well, though it may be better to be on the safer side here since email syncing is complex.